### PR TITLE
ecdsa instance builtin

### LINKED ISSUE #214: feat: enhancement of EC Operation Instance Definition implementation
## Description

The current EC Operation Instance Definition within our Cairo Zig codebase requires enhancements to align with standards and improve functionality: https://github.com/keep-starknet-strange/cairo-zig/blob/main/src/vm/types/ec_op_instance_def.zig

## Objectives

- Implement improvements in the EC Operation Instance Definition.
- Ensure comprehensive test coverage for validation and accuracy.

## References

- Reference to the Rust implementation: https://github.com/lambdaclass/cairo-vm/blob/8887e60f7ea5459188cb6624acc908ad29e75f60/vm/src/types/instance_definitions/ec_op_instance_def.rs
- Reference to the Python implementation: https://github.com/starkware-libs/cairo-lang/blob/51c287ac441ea0295775dce743470f87e395d4b1/src/starkware/cairo/lang/builtins/ec/instance_def.py

### LINKED ISSUE #215: feat: Ecdsa Instance Definition Enhancement
## Description
The current Ecdsa Instance Definition in our Cairo Zig codebase requires enhancements: [Ecdsa Instance Definition](https://github.com/keep-starknet-strange/cairo-zig/blob/main/src/vm/types/ecdsa_instance_def.zig).

## Objectives
- Implement improvements within the Ecdsa Instance Definition.
- Enhance test coverage for thorough validation and accuracy.

## Reference
- [Rust Implementation Reference](https://github.com/lambdaclass/cairo-vm/blob/8887e60f7ea5459188cb6624acc908ad29e75f60/vm/src/types/instance_definitions/ecdsa_instance_def.rs)

### PR DESCRIPTION
**Pull request by ptisserand**
Close #215

### CODE REVIEW

File: src/vm/types/ecdsa_instance_def.zig [hunk 1]
```diff
@@ -28,4 +34,54 @@ pub const EcdsaInstanceDef = struct {
             .n_hash_bits = 251,
         };
     }
+
+    pub fn _cells_per_builtin(_: Self) u32 {
```
Comment (tcoratger) [inline src/vm/types/ecdsa_instance_def.zig:38]: Following Ziglang naming convention can we rename this one like: `cellsPerBuiltin`?

File: src/vm/types/ecdsa_instance_def.zig [hunk 2]
```diff
@@ -28,4 +34,54 @@ pub const EcdsaInstanceDef = struct {
...
+        return CELLS_PER_SIGNATURE;
+    }
+
+    pub fn _range_check_units_per_builtin(_: *const Self) u32 {
```
Comment (tcoratger) [inline src/vm/types/ecdsa_instance_def.zig:42]: Following Ziglang naming convention can we rename this one like: `rangeCheckUnitsPerBuiltin`?

**Comment by tcoratger** [comment]
@ptisserand Thanks, following recent conversations we had internally, can you:
- Rename `init` to `initDefault`
- Rename `from` to `init`

Then we are good to merge (with my minor comments)

File: src/vm/types/ecdsa_instance_def.zig (hunk 2 above)
Comment (ptisserand) [reply src/vm/types/ecdsa_instance_def.zig]: Updated

**Comment by ptisserand** [comment]
> @ptisserand Thanks, following recent conversations we had internally, can you:
> 
> * Rename `init` to `initDefault`
> * Rename `from` to `init`
> 
> Then we are good to merge (with my minor comments)

Done

**Review by tcoratger — APPROVED** [review]

**Pull request merged**